packages/create-cloudflare/templates/next/c3.ts

```diff
@@ -1,45 +1,27 @@
-import { brandColor, dim } from "@cloudflare/cli/colors";
-import { spinner } from "@cloudflare/cli/interactive";
+import { brandColor } from "@cloudflare/cli/colors";
 import { runFrameworkGenerator } from "frameworks/index";
-import { readFile, usesTypescript, writeFile } from "helpers/files";
 import { installPackages } from "helpers/packages";
 import type { TemplateConfig } from "../../src/templates";
 import type { C3Context } from "types";
 
 const generate = async (ctx: C3Context) => {
-	await runFrameworkGenerator(ctx, [ctx.project.name]);
+	await runFrameworkGenerator(ctx, [
+		ctx.project.name,
+		// We are using the Cloudflare template for Next.js projects
+		// This is a template maintained by Cloudflare that is based on a standard Next.js starter project,
+		// and has already been configured to work with Cloudflare Workers
+		"--example",
+		"https://github.com/cloudflare/templates/tree/main/next-starter-template",
```

Comment:
What about moving the template to the OpenNext repo instead?

Reply:
That's definitely an option. The question is who will maintain it?
If we put it in OpenNext will it be kept up to date (at least more than than the Cloudflare templates repo)?
If there is no capacity to maintain it in Open Next then at least there is a team that is supposed to be on the hook to maintain the templates repo. I am told that this is a template they do intend to maintain.